afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/ImportedCaseBuilder.java

```diff
@@ -149,7 +149,7 @@ public ImportedCase build() {
         } catch (IOException e) {
             throw new UncheckedIOException(e);
         }
-
+        context.getStorage().consistent(info.getId());
         context.getStorage().flush();
 
         return new ImportedCase(new ProjectFileCreationContext(info, context.getStorage(), context.getProject()),
```

Comment:
Should we call `setConsistent` on the node info here?

Reply:
sure, we have to set the node info coherent attribute to true. that attributes let the node accessible when needed .